src/rpc/RPCHelpers.cpp

```diff
@@ -943,6 +956,28 @@ isFrozen(
     return false;
 }
 
+bool
+isDeepFrozen(
+    BackendInterface const& backend,
+    std::uint32_t sequence,
+    ripple::AccountID const& account,
+    ripple::Currency const& currency,
+    ripple::AccountID const& issuer,
+    boost::asio::yield_context yield
+)
+{
+    if (ripple::isXRP(currency))
+        return false;
+
+    if (issuer == account)
+        return false;
+
+    // Check if the account is deep frozen
+    return fetchAndCheckAnyFlagsExists(
```

Comment:
`ripple::keylet::account(issuer)` is an account object keylet. How this function can check ripplestate `lsfHighDeepFreeze` `lsfLowDeepFreeze` flag in account object ?